packages/edit-site/src/components/editor/index.js

```diff
@@ -153,6 +158,12 @@ export default function Editor() {
 		return <CanvasSpinner />;
 	}
 
+	const className = classnames( {
+		'show-icon-labels': showIconLabels,
+		'is-inserter-opened': isInserterOpen,
+		'is-list-view-opened': isListViewOpen,
```

Comment:
What do you think about using a single class name called e.g. `has-secondary-sidebar`? That way we're still maintaining the vague semblance of separation we have between the "interface skeleton" and the content shown within it.

Reply:
Yes makes sense to me. I was going to go with this first of all, but I hesitated because current classnames were based on what they represented (granted there were not many).